tests/acceptance-tests/test/specs/smoke-tests/simple-login.app-test.js

```diff
@@ -21,9 +23,9 @@ describe("simple login test", function () {
     const usrData = yaml.safeLoad(fs.readFileSync("./tests/acceptance-tests/config/user-data.yml", "utf8"));
 
     // default to process env if we've got that
-    const adminEmail = process.env.REACTION_EMAIL || usrData.admin_email;
-    const adminPassword = process.env.REACTION_AUTH || usrData.admin_pw;
-    const adminUserName = process.env.REACTION_USER || usrData.admin_user;
+    const adminEmail = process.env.REACTION_EMAIL;
```

Comment:
Please add those codes sir @Dannytebj

Reply:
He removed them only in his tests. It doesn't affect others